test/steps/v2_steps.py

```diff
@@ -858,6 +858,23 @@ def lookup_application(context, app_id):
     context.response = context.icl.applications(int(app_id))
 
 
+@when(
+    'we make a LookupApplicationLogsByID call with applicationID {app_id} limit {limit} minRound {min_round} maxRound {max_round} nextToken "{next_token:MaybeString}" sender "{sender:MaybeString}" and txID "{txid:MaybeString}"'
+)
+def lookup_application_logs(
+    context, app_id, limit, min_round, max_round, next_token, sender, txid
+):
+    context.response = context.icl.application_logs(
+        int(app_id),
+        limit=int(limit),
+        min_round=int(min_round),
+        max_round=int(max_round),
+        next_page=next_token,
```

Comment:
should we use next_token instead of next_page? next_page seems to imply an int value and next_token would be more consistent with the naming in go sdk.

Reply:
Other methods in this SDK also use `next_page`, so I did the same for consistency (but maybe we should change this in v2).